requirements.txt

```diff
@@ -28,7 +28,7 @@ django-tables2==2.4.1
 django-tinymce==3.4.0
 django-two-factor-auth==1.13.1
 django-webpack-loader==0.7.0
-django==2.2.26
+django==3.0.14
```

Comment:
The goal is Django 3.2.x (LTS). I assume you are taking it step by step to ease the upgrade.

Reply:
Yes, exactly. Its end goal is Django 3.2.